doc/source/api/visualization/graphics_windows_manager.rst

```diff
@@ -6,40 +6,39 @@ The ``GraphicsWindowsManager`` class provides for managing and directly interact
 with PyVista windows. By registering these methods with ``EventsManager``, you
 can update graphics during run time and create animations.
 
-The following example updates ``window-1`` during solution initialization and
-whenever data is read. During the calculation, it also updates ``window-1`` at
+The following example updates ``p_cont`` during solution initialization and
```

Comment:
@prmukherj Here we should just say the thing whatever it is that's being updated (which is somewhat automatic with better naming) so that the reader has some chance of following.